frontend/src/types/common.ts

```diff
@@ -598,6 +618,7 @@ export const empty = (
     project: EMPTY_PROJECT,
     dataSourceList: [],
     anomalyList: [],
+    labels: [],
```

Comment:
use xxxList instead of xxxS

Reply:
This is aligned to https://github.com/bytebase/bytebase/blob/main/api/database.go#L72

Reply:
OK, this is a bit unfortunate. Let's keep it as is then, will resolve it separately